include/config/rcxml.h

```diff
@@ -83,6 +83,7 @@ struct rcxml {
 
 	/* graphics tablet */
 	struct tablet_config {
+		enum rotation rotate;
```

Comment:
Suggest `rotation` instead of `rotate`.... so `enum rotation rotation`

Reply:
Adjusted, thanks!